src/wpt/fetch/api-test.ts

```diff
@@ -392,15 +392,8 @@ export default {
   'headers/headers-errors.any.js': {
     comment: 'Our validation of header names is too lax',
     expectedFailures: [
-      'Create headers giving bad header name as init argument',
```

Comment:
So we did end up getting a couple of WPT improvements out of it, nice.

Reply:
Yep, but we'll need to make sure this doesn't count as a breaking change that'll need a compat flag.

Reply:
Unfortunately, we're going to be stuck with these failures a bit longer. Kenton prefers the original `kj::parse` based approach and reverting away from the new lookup tables to the original brings back the errors, suggesting that there's a bug somewhere in the original validation table. As I suspected, tho, fixing it is likely to require a compat flag so we can't fix it this way anyway.